docs/proposed/psa-driver-developer-guide.md

```diff
@@ -36,6 +36,10 @@ A driver therefore consists of:
 
 Mbed TLS calls driver entry points [as specified in the PSA Cryptography Driver Interface specification](psa-driver-interface.html#driver-entry-points) except as otherwise indicated in this section.
 
+## mbedtls extensions
+
+The driver description can include mbedtls extensions, mbedtls extensions are meant to extend/help integrating the driver into the mbedTLS infrastructure.For example, **mbedtls/h_condition** can be used to conditionally include header files for a given driver. **mbedtls/c_condition** can be used to conditionally enable dispatch capabilities for a driver.
```

Comment:
> For example

This place is the reference documentation of Mbed TLS extensions, so it needs the full list and an explanation of what each is, to the same level of detail as the spec (so JSON type, whether it's mandatory, and what the values mean).

By the way, the official name of the library is spelled “Mbed TLS”.

Reply:
I addressed this, but if you want it reworded to suit mbed TLS literature, please go ahead, and I can have it reflected. 
I think with that we are mostly covered.